apps/backoffice-v2/src/common/components/molecules/RiskIndicator/RiskIndicator.tsx

```diff
@@ -37,14 +38,25 @@ export const RiskIndicator = ({
         {!!violations?.length &&
           violations.map(violation => (
             <li key={violation.label} className="flex list-none items-center text-slate-500">
-              <WarningFilledSvg
-                className={ctw('me-3 mt-px', {
-                  'text-slate-300 [&>:not(:first-child)]:stroke-background':
-                    violation.severity === 'low',
-                })}
-                width={'20'}
-                height={'20'}
-              />
+              {violation.severity !== Severity.LOW && (
```

Comment:
The gray part.
![image](https://github.com/ballerine-io/ballerine/assets/61207713/b6c2ef1e-da68-4819-bc25-3699832295ed)